redaxo/src/addons/media_manager/lib/effects/effect_image_properties.php

```diff
@@ -5,6 +5,8 @@
  */
 class rex_effect_image_properties extends rex_effect_abstract
 {
+    public const NO_INTERLACING = '- off -';
```

Comment:
Die braucht man eigentlich nicht von außen, oder?

```suggestion
    private const NO_INTERLACING = '- off -';
```